### ISSUE TITLE
Compability with GCCollab Wiki  ?

### ISSUE THREAD

**Issue by KingBain** [loogart/gcpedicure#3]
Hello,

I've copied the GCPedicure CSS template and GCPedicure Template over to a page on the GCCollab wiki. When I try to load the page though the page doesnt seem load the stylesheet.

![image](https://user-images.githubusercontent.com/367922/51485459-1fe0a400-1d6c-11e9-87f8-f82dd503797a.png)

**Comment by loogart** [comment]
@KingBain yes, no go with GCCollab Wiki. I guess there was a decision to limit the GCCollab visual freedom due to the nature of it being public facing.

I tried finding a quick solution during the Summer of 2018, but I didn't pursue it due to time restrictions.

**Comment by KingBain** [comment]
ill try and ask them :) I dont understand which part theyre blocking, but I think the mediawiki is blocking out the javascript includes

**Comment by loogart** [comment]
thanks!
